modules/contracts/apps/challenge/src/handlers/execute.rs

```diff
@@ -458,15 +456,13 @@ fn charge_penalty(deps: DepsMut, app: &ChallengeApp, challenge_id: u64) -> AppRe
     }
 
     let compute_amount_per_friend = || -> Result<u128, AppError> {
-        if challenge.total_check_ins == 0 || num_friends == 0 {
+        if num_friends == 0 {
             return Err(AppError::Std(StdError::generic_err(format!(
-                "Cannot compute amount per friend. total_check_ins: {}, num_friends: {}",
-                challenge.total_check_ins, num_friends
+                "Cannot compute amount per friend. num_friends: {}",
+                num_friends
```

Comment:
Cute how we use variable here, when we know the exact string here :rofl:

Reply:
Yeah, also the use of StdErrors here is bad. All of them should be replaced.